iml-gui/crate/src/components/action_dropdown/command_action_modal.rs

```diff
@@ -0,0 +1,90 @@
+use crate::{
```

Comment:
probably should just call this file `command_modal.rs`, and it should be directly under `/components` instead of inside `action_dropdown`.